pkg/bookmarks/bookmarks.go

```diff
@@ -25,12 +30,22 @@ func (b Bookmark) SSHInfoIsEmpty() bool {
 
 // ConvertToOptions returns an options struct from connection details
 func (b Bookmark) ConvertToOptions() command.Options {
+	user := b.User
```

Comment:
IMO we should fall back to env vars only if bookmark user is not set, not the other way around

Reply:
Hrm, that's fine by me but that would mean bookmark _password_ and user fallbacks differ. Since there's no way (afaict) to tell if a password is blank or unspecified, we'd still have to check passwordVar first.

Reply:
I meant to say that flow should be the same for both cases, ie if user/pass is set - use that first, otherwise get it from an env var.